pkg/features/feature.go

```diff
@@ -160,6 +166,25 @@ func InitializeFeatures(featuresClient managementv3.FeatureClient, featureArgs s
 	}
 }
 
+func SetFeature(featuresClient managementv3.FeatureClient, feature *Feature, value bool) error {
```

Comment:
Agreed here ^

Reply:
No response to this so far.

Reply:
https://github.com/rancher/rancher/pull/32814/files#diff-2f6213e2bd15bb256be4ed430e664c08612e86384445be797a20d5f56d48d101R184 It stills needs the ability to set in-memory map since it can be accessed in-memory. I just followed the code because it updates in both crd and in-memory struct.